app/models/vm_or_template.rb

```diff
@@ -461,30 +461,15 @@ def self.invoke_tasks_remote(options)
 
       begin
         raise "SOAP services are no longer supported.  Remote server operations are dependent on a REST client library."
```

Comment:
This section should not be deleted.  It is a temporary deletion until we add REST API support for cross-server tasks.  It is meant to come back before release.